war/src/main/scss/base/_layout-commons.scss

```diff
@@ -97,6 +97,12 @@ body.full-screen #main-panel {
   padding: 0;
 }
 
+@media (max-width: 1170px) {
```

Comment:
Given the next block
```
@media (width >= 1170px) {
  body.two-column #side-panel {
    width: 340px;
  }...
  }
 ```
 maybe just drop the media query here and remove the width setting from that other block?